Final pilot split update

cheeky-gorilla opened the pull request:
Update membership for May '23

cheeky-gorilla committed:
Update 9-membership.md

Update membership for May '23

jtraglia commented on docs/9-membership.md:
```diff
@@ -143,134 +157,148 @@ Read more about how weighting is determined in [**6.3 Weighting**](https://proto
+ | 0xD4a3030b5f5e8DD4860d370C17E8576aE9951a2D | 0.8778 |
+ | 0x10ab73AA48D686b7FD9ec9D50418a14DD23f6631 | 0.8778 |
+ | 0x6e22a5e30088c8389dc725bbead5f0675334299f | 0.8873 |
+ | 0x24113fFB07189D1e6E169025A424B58C29522972 | 0.9152 |
```
Why are the numbers on 0xSplits slightly different? Is 0xSplits out-of-date right now?

Picking one at random, 0x2411... is 0.9152 here but 0.906% on 0xSplits:

<img width="302" alt="image" src="https://github.com/protocolguild/docs/assets/95511699/dcf2c380-df16-4001-93fe-f2db0454f17b">

cheeky-gorilla replied:
Yep 0xSplits is out of date, since it's a snapshot of the weights at the time the last membership update was made onchain. 

Whereas my changes reflect what we have in the Airtable, which is automatically calculated based on today's date! So it's technically more accurate, but  I dont think that's worth the potential confusion. So I'll edit to use 0xSplits as the source instead of Airtable. Thanks Justin!

jtraglia replied:
Okay cool, thanks 😄

cheeky-gorilla committed:
Final pilot split update attempt #2

Update membership for May '23, with weights reflecting 0xSplits not Airtable to avoid confusion.

tvanepps committed:
Merge pull request #46 from protocolguild/cheeky-gorilla-patch-2

Final pilot split update attempt #2

tvanepps commented:
replaced by https://github.com/protocolguild/docs/pull/46
